packages/snap/src/core/services/subscriptions/WebSocketConnectionService.ts

```diff
@@ -332,4 +353,84 @@ export class WebSocketConnectionService {
       connectionRecoveryHandlers: this.#connectionRecoveryHandlers,
     });
   }
+
+  async #handleOnActive(): Promise<void> {
+    this.#logger.log(`Client became active`);
+
+    // Cancel any existing close connections background event
+    await this.#cancelCloseConnectionsBackgroundEvent();
+
+    // Open the connections
+    await this.#openConnectionsForActiveNetworks();
```

Comment:
This is not opening them again if they are already open right? Just double checking